pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -109,6 +109,10 @@ type KeycloakAPIRealm struct {
 	// Require SSL
 	// +optional
 	SslRequired string `json:"sslRequired,omitempty"`
+
+	// Email
```

Comment:
Just a nitpick. Maybe "SMTP server" would be more descriptive here?

Reply:
All settings of course relate to the SMTP server in one way or another, however since it also includes things like From and display name, I'd like to stick to the more general Admin UI labelling `Email`.